### PR TITLE
feat(debian): add buster support

### LINKED ISSUE #276: Does not work with Debian Buster
Debian buster fails because it's not added to the yaml files.

### PR DESCRIPTION
**Pull request by johnkeates**
feat(debian): add buster support

This adds support for Debian buster and native package installation. Fixes #276

### CODE REVIEW

File: postgres/codenamemap.yaml [hunk 1]
```diff
@@ -54,12 +54,14 @@
...
 {{ debian_codename('jessie', '9.4') }}
 {{ debian_codename('stretch', '9.6') }}
+{{ debian_codename('buster', '11') }}
 
 # `oscodename` grain has long distro name
 # if `lsb-release` package not installed
 {{ debian_codename('wheezy', '9.1', 'Debian GNU/Linux 7 (wheezy)') }}
 {{ debian_codename('jessie', '9.4', 'Debian GNU/Linux 8 (jessie)') }}
 {{ debian_codename('stretch', '9.6', 'Debian GNU/Linux 9 (stretch)') }}
+{{ debian_codename('buster', '11', 'Debian GNU/Linux 10 (buster)') }}
```
Comment (aboe76) [inline postgres/codenamemap.yaml]: shouldn't this be '10' ?

Comment (johnkeates) [reply postgres/codenamemap.yaml]: No, that's a mistake I made as well :p this adds the PostgreSQL version that exists in the distro repo and doesn't refer to the distrio version at all.

**Comment by johnkeates** [comment]
Travis doesn't seem to like my commit message for some reason. But the output doesn't really give me a pointer to what's wrong:

✖   subject may not be empty [subject-empty]
✖   type may not be empty [type-empty]

**Comment by johnkeates** [comment]
Thanks @noelmcloughlin should be all better now

File: postgres/codenamemap.yaml (hunk 1 above)
Comment (aboe76) [reply postgres/codenamemap.yaml]: thanks for pionting this out.

**Comment by noelmcloughlin** [comment]
> should be all better now

No, it's the commit message you need to change. ;-)

**Comment by johnkeates** [comment]
> > should be all better now
> 
> No, it's the commit message you need to change. ;-)

I really should RTFM :-D

**Pull request merged**